build-java/build.gradle

```
@@ -0,0 +1,27 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "org.openapitools:openapi-generator-gradle-plugin:4.0.0-beta2"
+    }
+}
+
+apply plugin: 'org.openapi.generator'
+
+def clientOutputDir = "$projectDir/../generated-clients/java-clients".toString()
```

There is client in the parent dir, rename java-clients to java. Also it follows the same naming as OpenAPITools generator.
```suggestion
def clientOutputDir = "$projectDir/../generated-clients/java".toString()
```